Netw4ppl/app/src/main/java/netw4ppl/ines/utils/AdapterViewFields.java

```diff
@@ -293,10 +293,10 @@ public AdapterViewFields(Context context, ArrayList<Field> fields, boolean new_p
      * @param viewType an int representing the view type
      * @return a RecyclerView.ViewHolder object. The type of the object depends of the input type specified.
      */
-    @NonNull
     @Override
     public RecyclerView.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
         View view;
+        System.out.println("viewType = " + viewType);
```

Comment:
TO be deleted before merging to preprod. Maybe you should add a tag to remember

Reply:
👍🏼